config/dev.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "MASQ_APP_BASE_URL": "http://localhost:3000"
+}
```

Comment:
You should  configure your editor to add newline

Reply:
done :+1: